eslint-config-angular/package.json

```
@@ -32,7 +32,7 @@
   "license": "MIT",
   "peerDependencies": {
     "@eslint/js": "^9.9.1",
-    "angular-eslint": "^18.1.0 || ^19.0.0 || ^20.0.0",
+    "angular-eslint": "^19.0.0 || ^20.0.0 || ^21.0.1",
```

This is considered a breaking change, since we'll no longer support projects with angular-eslint `^18.1.0`. Please adapt the commit message accordingly to mention this.

Additionally, we usually allow the whole range of each major version, hence I don't see a reason why we shouldn't support `21.0.0` here.

```suggestion
    "angular-eslint": "^19.0.0 || ^20.0.0 || ^21.0.0",
```

@spike-rabbit WDYT?